routes/api/index.js

```diff
@@ -23,6 +23,15 @@ router.use('/users', config.DEV_MODE ? function(req,res,next){
     return res.status(500).send("could not create the user");
   });}:passport.authenticate('bearer'), require('./users'));
 
+app.get('*', (req, res) => {
+  res.status(404).json({
+    "error": "404",
```

Comment:
a json cannot have two keys of the same name. like javascript 101 my friend. 

please at least run your code before you make PRs